docs/_sidebar.md

```diff
@@ -16,7 +16,8 @@
   - [Work on Mobile app](how-to-setup-freecodecamp-mobile-app-locally.md)
   - [Work on tutorials with CodeRoad](how-to-work-on-tutorials-that-use-coderoad.md)
   - [Work on Localized Web App](how-to-work-on-localized-client-webapp.md)
-  - [Work on Cypress tests](how-to-add-cypress-tests.md)
+  - [Work on Cypress Tests](how-to-add-cypress-tests.md)
```

Comment:
You know what? Let's deprioritize this and move to the additional guides section while we are updating the sidebar.